src/pages/dashboard/_CDCComparisonContainer.js

```diff
@@ -42,6 +42,7 @@ export default function CDCComparisonContainer() {
       }
     })
     .sort(sortChronologically)
+    .filter(element => element.date > new Date(2020, 2, 3))
```

Comment:
Hmm, I think this is somewhat fragile. Hardcoding the date assumes that earlier Covid Tracking data doesn't become available. I also noticed that the current data set for CDC ends before the Covid Tracking data... which creates mismatches on the right end of the time scale.

I'm working on a solution that normalizes things. It would even work if older Covid Tracking data comes to light.